### PR TITLE
video page of video library

### PR DESCRIPTION
**Pull request by devAmlan**
This is the video library of my Mlan tube app

1. User can see a video listing page where all the videos are listed.
2. A filter of video category

### CODE REVIEW

**Comment by devAmlan** [comment]
mlan-tube-videos

**Review by srishtigautam21 — APPROVED** [review]
Easily understandable and clean code
Appreciation:
Through the short video your UI looks amazing
Good work 
Good to merge

File: src/components/navbar/Navbar.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  margin-left: 2rem;
+  color: var(--primarycolor);
+}
+.search_bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.search_bar input {
+  width: 27rem;
```
Comment (dev-prabhat) [inline src/components/navbar/Navbar.css]: you can also use percentage to provide width with respect to your header container

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
Easy to read code
UI is clean
Suggested some changes

File: src/components/navbar/Navbar.css (hunk 1 above)
Comment (devAmlan) [reply src/components/navbar/Navbar.css]: Thanks for the suggestion @dev-prabhat , I will use %

**Pull request merged**